### ISSUE TITLE
Consider adding a license to make this project open source

### ISSUE THREAD

**Issue by mbbroberg** [angsamuel/GingerVR#1]
Hi @angsamuel, a Correspondent for Opensource.com wants to write about this project and we only cover open source projects. I'm wondering if you would like to make this repository an officially open source project by adding a license reviewed by [the OSI](https://opensource.org/osd-annotated). Doing so is a great way to make the terms of use clear and to provide your users with the freedom to copy, modify, and reproduce it legally around the world.

If you're looking for a quick review of options, check out https://choosealicense.com/licenses/. 

TL;DR MIT is a good default since it allows reuse with very few terms. GPL is great if you're passionate about [forks remaining open source as well](https://www.gnu.org/philosophy/free-sw.en.html).

**Comment by angsamuel** [comment]
Hey @mbbroberg This is definitely something we need to add. I'll be updating the project's readme tomorrow and I'll throw a license up in addition to that.

**Comment by mbbroberg** [comment]
That's great news. I'll keep this project on our news roundup list and wait for the license to be added. Thanks again for sharing your hard work with us all.
